src/amo/searchUtils.js

```diff
@@ -9,6 +9,12 @@ import {
 import log from 'amo/logger';
 import { USER_AGENT_OS_IOS } from 'amo/reducers/api';
 
+export function generate_threshold_params({ param }) {
```

Comment:
```suggestion
export function generateThresholdParams({ param }) {
```

Reply:
We probably don't need to use an object as first and only argument, `param` only should be enough.